interface/eRx.php

```diff
@@ -121,7 +121,7 @@ function array_key_exists_default($key, $search, $default = null)
 
         if (count($errors) > 0) {
             ?>
-        <strong><?php echo xlt('NewCrop call failed'); ?></strong>
+        <strong><?php echo xlt('Ensora call failed'); ?></strong>
```

Comment:
Hmm, I think we need to be careful changing translated strings. If there's already a translation of 'NewCrop call failed' somewhere, changing 'NewCrop' to 'Ensora' will break the lookup. We should probably figure out a way to avoid proper nouns in our translations, for the most part.

I'd be curious about @bradymiller 's take here, since I think this probably goes mostly unnoticed.

Reply:
Hmm, this makes me wonder if we use BiDi markers properly to handle not-transliterated proper names when going from a LTR to a RTL language.